tinyasm/tinyasm.cpp

```diff
@@ -385,40 +384,43 @@ PYBIND11_MODULE(_tinyasm, m) {
     PetscLogEventRegister("PCTinyASMApply", PC_CLASSID, &PC_tinyasm_apply);
     m.def("SetASMLocalSubdomains",
           [](PC pc, std::vector<IS> ises, std::vector<PetscSF> sfs, std::vector<PetscInt> blocksizes, int localsize) {
-              PetscInt i, p, numDofs;
-              PetscCall(PetscLogEventBegin(PC_tinyasm_SetASMLocalSubdomains, pc, 0, 0, 0));
+              PetscInt p, numDofs;
+
+              MPI_Comm comm = PetscObjectComm((PetscObject) pc);
+
+              PetscCallAbort(comm, PetscLogEventBegin(PC_tinyasm_SetASMLocalSubdomains, pc, 0, 0, 0));
```

Comment:
We need `PetscCallAbort` because `PetscCall` and `PetscCallVoid` both `return` something, which messes up pybind11.